src/Raven.Studio/typescript/viewmodels/resources/widgets/widget.ts

```diff
@@ -110,6 +110,10 @@ abstract class widget<TConfig = unknown, TState = unknown> {
     restoreConfiguration(config: TConfig) {
         // empty by default
     }
+
+    openWidgetSettings(): void {
```

Comment:
do we call that from abstract class? Do we need this method in parent class?

Reply:
removed it from parent